Sources/NSAttributedStringExtensions.swift

```diff
@@ -73,5 +73,12 @@ public func += (left: inout NSAttributedString, right: NSAttributedString) {
     ns.append(right)
     left = ns
 }
+    
+/// EZSE: Sum of one NSAttributedString with another NSAttributedString
+public func + (left: NSAttributedString, right: NSAttributedString) -> NSAttributedString {
```

Comment:
Its not really about adding something. I like the extension, I am just confirmed about operator precedence. 

Ideally + should be higher precedence than +=.

For instance str1 += str2 + str3 should evaluate to str1 = (str1 + (str2 + str3))). I do not want the assignment to be done prior to the +. 

I wanted to verify that this behavior is consistent for this.